src/components/charts/IndexBarChart/index.jsx

```diff
@@ -125,8 +120,8 @@ class IndexBarChart extends React.Component {
           >
             <h4>Project Submission status</h4>
             <XAxis
-              stroke={localTheme['barGraph.lineColor']}
-              fill={localTheme['barGraph.lineColor']}
+              stroke='#666666'
+              fill='#666666'
```

Comment:
is there a way to make these configurable, like pass a prop to the chart?

Reply:
I think for now we only have 1 style (with white background), it is not necessary to make text colors configurable...

Reply:
true, but if we want to use this chart anywhere else in the future we'd want it to be a prop not hardcoded